lib/server.js

```diff
@@ -108,19 +108,25 @@ exports.create = function create(createOptions) {
 
     try {
       if (this.useRouter) {
-        match(thing, function (error, redirectLocation, renderProps) {
+        match({ routes:createOptions.routes, location:thing},
+          function(error, redirectLocation, renderProps) {
           if (redirectLocation) {
+            debug('server.js match 302 %s', redirectLocation);
```

Comment:
thanks. I think for 404 and 500 its a great idea to have the user pass in a component. But 302 is tricky, it really needs to go and hit the browser. With no `res` object inside the renderer its getting complicated to handle that.